portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,26 +24,31 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.ws.rs.core.MediaType;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   private Gson gson = new Gson();
-
+  private ArrayList<Comment> comments = new ArrayList<Comment>();
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType(MediaType.APPLICATION_JSON);
-    ArrayList<Comment> comments = new ArrayList<Comment>();
-    Comment comment1 = new Comment("1","Ada","So cool!");
-    Comment comment2 = new Comment("2","G","Don't be evil");
-    comments.add(comment1);
-    comments.add(comment2);
-    String json = commentsToJson(comments);
+    String json = commentsToJson();
     response.getWriter().println(json);
   }
 
-  private String commentsToJson(ArrayList<Comment> comments){
-    String json = this.gson.toJson(comments);
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      int id = this.comments.size();
```

Comment:
For now it will work - every comment would have an unique id.
But what would happen in the future, is some comments are deleted (for example by a moderator)? THen you could end up with duplicated ids, e.g.:
1. First comment added (id 0, since there is no comments yet)
2. Second comment added (id 1)
3. First comment deleted. Now we have only [1] in the list of comments.
4. New commend added. It would receive id 1 again!

Suggestion: store private counter that would increase every time a comments is added.

Reply:
<trying to start a review>

Reply:
Yes, I agree, this was just an MVP. Right now I am using the id generated by datastore, which I think is guaranteed to be unique

Reply:
By "right now" you mean in this Pull Request? I don't see any changes. Or you mean in some work-in-progress code?